### PR TITLE
Add mobile layout

### PR DESCRIPTION
**Pull request by dvcorreia1984**
Added the following in this pull request: 

1. Mobile layout for home page and about page
2. Desktop layout with media queries
3. Javascript to add actions to buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,46 +6,150 @@
...
-    <h3>"Hello! Sharing world"</h3>
-    <h1>CREATIVE COMMONS GLOBAL SUMMIT 2023</h1>
-    <div class="textbox">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</div>
-    <h2>2015.10.15(THU)~16(FRI)</h2>
-    <p>@ National Museam of Karea, Art Center Nabi and more.." </p>
+    <div id="textcontainerhome1">
+    <h3 id="spacingtop">"AI, friend or foe?"</h3>
+    <h1 id="h1desk">ARTIFICIAL INTELLIGENCE GLOBAL SUMMIT 2023</h1>
+    <div class="textbox">How should we plan ahead for the disruption that AI will bring to our world? Delegates from over 20 countries will be sharing their expertise with us. We look forward to hosting you in Cape Town, South Africa for this unique event.</div>
+    <h2 class="contdesk">2023.10.15(THU) ~ 16(FRI)</h2>
```
Comment (Lucas-Masaba) [inline index.html:63]: - [ ] This text is too large compared to that in the original design. Please scale it down and add space between in and the next section as seen in the original design
### Your Implementation
image
### Required
image

File: index.html [hunk 2]
```diff
@@ -6,46 +6,150 @@
...
+
+    </div>
 </section>
 
+<section id="home4">
+    <div class="container cont7 cont9" >
+    <h2 id="h2about4">Partner</h2>
+    <div id="linehome4" class="line"></div>
+    <div class="partner">
+        <ul class="brands">
```
Comment (Lucas-Masaba) [inline index.html:128]: - [ ] When the screen size is at 100%, your brand logos overlap their container. Please position these as seen in the original design.
### Your Implementation
image
### Required
image

File: README.md [hunk 3]
```diff
@@ -244,48 +138,30 @@ Feel free to check the [issues page](../../issues/).
...
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+I would like to thank the follolowing: 
 
-I would like to thank...
+- [Microverse](https://www.microverse.org/)
+- [Cindy Chin - as this project is based on her design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (Lucas-Masaba) [inline README.md]: - [ ]  Instead of "Cindy Chin", it should be "Cindy Shin". Please correct that

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @dvcorreia1984  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Use of GitHub flow :surfer: 
- Professional README :scroll: 
- Most features were implemented :gear: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (dvcorreia1984) [reply index.html:63]: Ok, I think I managed to fix this.

File: README.md (hunk 3 above)
Comment (dvcorreia1984) [reply README.md]: Fixed this.

File: index.html (hunk 2 above)
Comment (dvcorreia1984) [reply index.html:128]: Ok, I improved my version.

**Review by alexander16108 — APPROVED** [review]
Hi @dvcorreia1984 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linter checks passing ✔️
- Use of GitHub flow 🏄
- Professional README 📜
- Most features were implemented ⚙️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**